### DISCUSSION TITLE
Welcome to HZMZ Discussions!

### DISCUSSION THREAD

**Discussion by HZMusicDAO** [HZMusicDAO/HZMZ#1] [Announcements]
-->
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by PhantomDia** [comment]
Hey,

I'm the HZ Music founder/inventor and I'm so excited to be building this project with such a diverse, talented, amazing team. As we build the community, I want to remain accessible and open to create the best music platform possible!

Feel free to share feedback, ideas, suggestions, or requests for support.

Thanks,

GSG
